### PR TITLE
Task done: Embedded screenshots in markdown description(API for save images).

### LINKED ISSUE #1206: Task: API for saving images
In order to implement the insert image feature in description formatting, create an API which will accept an image and return its URL.

image

### PR DESCRIPTION
**Pull request by kr-2003**
Fixed #1206 

Added the functionality of embedding screenshots in markdown description in the form of ```$[img1]```,  ```$[img2]```, ```$[img3]```, ```$[img4]```, and ```$[img5]```.

Also added description section in issue page.(Wasn't added in the first place).

https://github.com/OWASP/BLT/assets/96587705/7e75e766-8c2c-4dd1-a7c9-6a604c6ad3ad

### CODE REVIEW

**Comment by kr-2003** [comment]
@DonnieBLT Kindly review this PR.

File: website/templates/issue.html [hunk 1]
```diff
@@ -426,14 +428,41 @@
         {% endif %}
     </div>
 </div>
-
+<script src="https://cdnjs.cloudflare.com/ajax/libs/showdown/1.9.1/showdown.min.js"></script>
```
[the remark this answers is not in the document]
Comment (DonnieBLT) [reply website/templates/issue.html:431]: > ## Inclusion of functionality from an untrusted source
> 
> Script loaded from content delivery network with no integrity check.
> 
> [Show more details](https://github.com/OWASP/BLT/security/code-scanning/98)

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
Please see the security alert. Thank you for including the video.

File: website/templates/issue.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (kr-2003) [reply website/templates/issue.html]: Added integrity check.

**Comment by kr-2003** [comment]
@DonnieBLT  I added the integrity check when including a cdn. Is there any other changes need to be done?

**Review by DonnieBLT — APPROVED** [review]

**Comment by DonnieBLT** [comment]
Nice work, thank you!

**Pull request merged**